src/forecasts.jl

```diff
@@ -120,6 +120,11 @@ function get_window_common(
     end
 
     data = get_data(forecast)[initial_time]
+    if length(size(data)) == 3
+        @assert size(data)[1] == 1
+        data = data[1, :, :]
```

Comment:
Why is this necessary?  A comment would be helpful.  This makes a copy of the data.  You could probably use `@view` instead.